pages/linux/dbus-daemon.md

```diff
@@ -0,0 +1,31 @@
+# dbus-daemon
+
+> The D-Bus message daemon, allowing multiple programs to exchange messages.
+
+- Run the daemon with a configuration file:
+
+`dbus-daemon --config-file {{path/to/file}}`
+
+- Run the daemon with the standard per-login-session message bus configuration:
+
+`dbus-daemon --session`
+
+- Run the daemon with the standard systemwide message bus configuration:
+
+`dbus-daemon --system`
+
+- Set the address to listen on and ignore the configuration for this:
```

Comment:
What does "this" mean here ? Do you mean the configuration file ?

Reply:
"this" is referring to the configuration setting inside the config file. There should be a value in the file such as `<listen>unix:path=/tmp/foo</listen>`. I guess maybe `and override any value in the configuration file`? Or something like that.

Reply:
Ah ok, I think it maybe be better written as - `Set the address to listen on and override the configuration value for it.`